### PR TITLE
One connector to rule them all

### PR DESCRIPTION
**Pull request by dacrome**

### CODE REVIEW

File: src/test/java/org/osiam/client/AbstractIntegrationTestBase.java [hunk 1]
```diff
@@ -43,36 +45,29 @@
...
-        OsiamConnector.Builder oConBuilder = new OsiamConnector.Builder()
-                .setAuthServerEndpoint(AUTH_ENDPOINT_ADDRESS)
-                .setResourceServerEndpoint(RESOURCE_ENDPOINT_ADDRESS)
-                .setClientId("short-living-client")
-                .setClientSecret("other-secret");
-        oConnector = oConBuilder.build();
-        accessToken = oConnector.retrieveAccessToken("marissa", "koala", Scope.ALL);
+    protected AccessToken accessToken;
+
+    protected void retrieveMarissaAccessToken() {
```
Comment (tkrille) [inline src/test/java/org/osiam/client/AbstractIntegrationTestBase.java]: Suggestion for method name: `retrieveAccessTokenForMarissa`

Comment (tkrille) [inline src/test/java/org/osiam/client/AbstractIntegrationTestBase.java]: Is this method really used everywhere it could be used?

Comment (dacrome) [inline src/test/java/org/osiam/client/AbstractIntegrationTestBase.java]: > Is this method really used everywhere it could be used?

Sorry?

Comment (dacrome) [inline src/test/java/org/osiam/client/AbstractIntegrationTestBase.java]: updated

Comment (tkrille) [inline src/test/java/org/osiam/client/AbstractIntegrationTestBase.java]: What I mean is, that you created this method to retrieve the "vanilla" access token for `marissa` that's used almost everywhere in the tests. But I think there are still some places where you don't call this method, but actually retrieve the access token like `connector.retrieveAccessToken(...)`.

Comment (dacrome) [inline src/test/java/org/osiam/client/AbstractIntegrationTestBase.java]: Yes, but mostly with a different configuration like in [ScopeIT](https://github.com/dacrome/connector4java-integration-tests/blob/one-connector/src/test/java/org/osiam/client/ScopeIT.java#L72).

**Comment by dacrome** [comment]
updated.

**Pull request merged**